backend/linux_vt.h

```diff
@@ -0,0 +1,67 @@
+/*
+ * Twin - A Tiny Window System
+ * Copyright (c) 2024 National Cheng Kung University, Taiwan
+ * All rights reserved.
+ */
+
+#ifndef _LINUX_VT_H__
+#define _LINUX_VT_H__
+
+#include <fcntl.h>
+#include <linux/kd.h>
+#include <linux/vt.h>
+#include <stdlib.h>
+#include <sys/ioctl.h>
+#include "twin_private.h"
+
+static inline int twin_vt_open(int vt_num)
+{
+    int fd;
+
+    char vt_dev[30] = {0};
+    snprintf(vt_dev, 30, "/dev/tty%d", vt_num);
+
+    fd = open(vt_dev, O_RDWR);
+    if (fd < 0) {
+        log_error("Failed to open %s", vt_dev);
+    }
+
+    return fd;
+}
+
+static inline int twin_vt_set(int fd, int mode)
```

Comment:
It is confusing to have similar function names, `twin_vt_set` and `twin_vt_setup`. Can you clarify?

Reply:
Yes, the clarity of this function needs to be improved. Here are a few options for this function name:
1. twin_vt_apply_mode
2. twin_vt_configure_mode
3. twin_set_vt_mode

Or let me know the better choices. I will rename this function.